wpengine/faustjs
# Feat: Update block-support-example docs

## Tasks

- [] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.

## Description

Issue when running block-support example with a 500 HTTP status error.

After debugging the issue with enabling logging for the operations.query I found `Type loader is expected to return a callable or valid type "CreateBlockBlockB`

However I missed on crucial step which was to run `npm run blockset` to install the block in WordPress.

I have now updated the README for block-support-example


## Related Issue(s):



## Testing

1. Setup latest WP with Local and latest WP GraphQL, Faust and content blocks
2. Cloned the latest canaray branch for faust
3. Ran Faust from the block-support directory after adding an .env.local
4. See screeenshots below


## Screenshots

<img width="771" alt="image" src="https://github.com/user-attachments/assets/a2fcaf82-d0e2-4659-8aa9-0bfcd5586eb9" />

<img width="1266" alt="image" src="https://github.com/user-attachments/assets/5e7d2e7d-1037-4a2e-a9fd-bbdae5ba6da0" />


## Documentation Changes



## Dependant PRs
